### ISSUE TITLE
Allow setting of Chocolatey GUI settings via Chocolatey package parameter

### ISSUE THREAD

**Issue by gep13** [chocolatey/ChocolateyGUI#592]
At the minute, it is possible to set a preference for whether Tile View is used or not via the Settings screen within Chocolatey GUI.

However, it should be possible to set this at installation time.

This will be done via a package parameter.

**Comment by gep13** [comment]
This will include all settings, i.e:

* Show Console Output
* Default to Tile View for Local Source
* Default to Tile View for Remote Source
* Use Delayed Search
* Allow non-admin access to Settings (C4B)

**Comment by exportgoldman** [comment]
Awesume. I vote for this feature! Currently we are coping a blank db with the settings we want to each PC. Ugly!

**Comment by gep13** [comment]
@exportgoldman I have just pushed a PR that starts to implement this functionality.  Would you be in a position to take this for a spin to get some feedback on it?

**Comment by gep13** [comment]
@exportgoldman The PR has now been merged, and it should now be possible to install the latest version from the MyGet feed in order to take this for a spin.  There is still a small amount of work to be done to add Package Parameters to the Chocolatey GUI package so that they can be passed in at installation time, however, for now, once installed, you should be able to run the following commands:

`chocolateygui feature -h` - to get the help documentation for the command
`chocolateygui feature list` - to get the current status of all the Chocolatey GUI features
`chocolateygui feature enable -n=<featureName>` - to enable a given feature
`chocolateygui feature disable -n=<featureName>` to disable a given feature

Let me know if you have any questions.

**Comment by exportgoldman** [comment]
Okay, I can only find the 0.16 build from the feed.


I try to run


chocolateygui feature list


and I get the chocolatey GUI appear, then a window appear which what looks like the options then half a second later it all quits.

________________________________

**Comment by exportgoldman** [comment]
Okqy ignore the last message. I have the beta version now and can see the options. This is great.

________________________________

**Comment by gep13** [comment]
@exportgoldman sweet!  Let me know if you have any questions.

**Comment by gep13** [comment]
The ability to run an installation command similar to the following:

```
choco install chocolateygui --params="'/ShowConsoleOutput=$true /UseDelayedSearch=$false /OutputPackagesCacheDurationInMinutes=120'
```

has just been added to the Chocolatey GUI preview, and it will be shipped in the next release of Chocolatey GUI (0.17.0).
